Synthesis failed for nodejs-pubsub

Hello! Autosynth couldn't regenerate nodejs-pubsub. :broken_heart:

Here's the output from running `synth.py`:

```
..

> @google-cloud/pubsub@2.3.0 compile /home/kbuilder/.cache/synthtool/nodejs-pubsub
> tsc -p . && cp -r protos build/

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m


Found 22 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/pubsub@2.3.0 compile: `tsc -p . && cp -r protos build/`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/pubsub@2.3.0 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-07-30T17_42_45_281Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/pubsub@2.3.0 prepare: `npm run compile-protos && npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/pubsub@2.3.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-07-30T17_42_45_304Z-debug.log
2020-07-30 10:42:45,320 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
2020-07-30 10:42:45,339 synthtool [DEBUG] > Wrote metadata to synth.metadata.
DEBUG:synthtool:Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-pubsub/synth.py", line 25, in <module>
    node.postprocess_gapic_library()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 193, in postprocess_gapic_library
    install(hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-07-30 10:42:45,381 autosynth [ERROR] > Synthesis failed
2020-07-30 10:42:45,381 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 901b355 chore(node): fix kokoro build path for cloud-rad (#1073)
2020-07-30 10:42:45,405 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-07-30 10:42:45,425 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing node_modules/
Traceback (most recent call last):
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 367, in synthesize_loop
    synthesize_inner_loop(fork, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 411, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 657, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 506, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 637, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    synthesize_loop_single_pr(toolbox, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 401, in synthesize_loop_single_pr
    synthesize_inner_loop(toolbox, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 411, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/b205fd33-200c-4298-88b8-18b0d1c79a3e/targets/github%2Fsynthtool;config=default/tests;query=nodejs-pubsub;failed=false).

Autosynth is still having trouble generating nodejs-pubsub. :sob:

Here's the output from running `synth.py`:

```
..

> @google-cloud/pubsub@2.3.0 compile /home/kbuilder/.cache/synthtool/nodejs-pubsub
> tsc -p . && cp -r protos build/

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m


Found 22 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/pubsub@2.3.0 compile: `tsc -p . && cp -r protos build/`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/pubsub@2.3.0 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-07-31T18_41_55_823Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/pubsub@2.3.0 prepare: `npm run compile-protos && npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/pubsub@2.3.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-07-31T18_41_55_847Z-debug.log
2020-07-31 11:41:55,860 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
2020-07-31 11:41:55,877 synthtool [DEBUG] > Wrote metadata to synth.metadata.
DEBUG:synthtool:Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-pubsub/synth.py", line 25, in <module>
    node.postprocess_gapic_library()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 193, in postprocess_gapic_library
    install(hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-07-31 11:41:55,922 autosynth [ERROR] > Synthesis failed
2020-07-31 11:41:55,922 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 901b355 chore(node): fix kokoro build path for cloud-rad (#1073)
2020-07-31 11:41:55,945 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-07-31 11:41:55,964 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing node_modules/
Traceback (most recent call last):
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 367, in synthesize_loop
    synthesize_inner_loop(fork, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 411, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 657, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 506, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 637, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    synthesize_loop_single_pr(toolbox, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 401, in synthesize_loop_single_pr
    synthesize_inner_loop(toolbox, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 411, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/7b446397-88f3-4463-9e7d-d2ce7069989d/targets/github%2Fsynthtool;config=default/tests;query=nodejs-pubsub;failed=false).

Autosynth is still having trouble generating nodejs-pubsub. :sob:

Here's the output from running `synth.py`:

```
..

> @google-cloud/pubsub@2.3.0 compile /home/kbuilder/.cache/synthtool/nodejs-pubsub
> tsc -p . && cp -r protos build/

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m


Found 22 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/pubsub@2.3.0 compile: `tsc -p . && cp -r protos build/`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/pubsub@2.3.0 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-01T17_00_19_723Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/pubsub@2.3.0 prepare: `npm run compile-protos && npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/pubsub@2.3.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-01T17_00_19_748Z-debug.log
2020-08-01 10:00:19,765 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
2020-08-01 10:00:19,786 synthtool [DEBUG] > Wrote metadata to synth.metadata.
DEBUG:synthtool:Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-pubsub/synth.py", line 25, in <module>
    node.postprocess_gapic_library()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 193, in postprocess_gapic_library
    install(hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-08-01 10:00:19,846 autosynth [ERROR] > Synthesis failed
2020-08-01 10:00:19,846 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 901b355 chore(node): fix kokoro build path for cloud-rad (#1073)
2020-08-01 10:00:19,873 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-08-01 10:00:19,893 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing node_modules/
Traceback (most recent call last):
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 367, in synthesize_loop
    synthesize_inner_loop(fork, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 411, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 657, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 506, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 637, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    synthesize_loop_single_pr(toolbox, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 401, in synthesize_loop_single_pr
    synthesize_inner_loop(toolbox, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 411, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/77487cc5-a367-44ee-984f-71b13efe0194/targets/github%2Fsynthtool;config=default/tests;query=nodejs-pubsub;failed=false).

Autosynth is still having trouble generating nodejs-pubsub. :sob:

Here's the output from running `synth.py`:

```
..

> @google-cloud/pubsub@2.3.0 compile /home/kbuilder/.cache/synthtool/nodejs-pubsub
> tsc -p . && cp -r protos build/

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m


Found 22 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/pubsub@2.3.0 compile: `tsc -p . && cp -r protos build/`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/pubsub@2.3.0 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-02T15_58_23_858Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/pubsub@2.3.0 prepare: `npm run compile-protos && npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/pubsub@2.3.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-02T15_58_23_880Z-debug.log
2020-08-02 08:58:23,896 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
2020-08-02 08:58:23,915 synthtool [DEBUG] > Wrote metadata to synth.metadata.
DEBUG:synthtool:Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-pubsub/synth.py", line 25, in <module>
    node.postprocess_gapic_library()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 193, in postprocess_gapic_library
    install(hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-08-02 08:58:23,955 autosynth [ERROR] > Synthesis failed
2020-08-02 08:58:23,955 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 901b355 chore(node): fix kokoro build path for cloud-rad (#1073)
2020-08-02 08:58:23,979 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-08-02 08:58:23,998 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing node_modules/
Traceback (most recent call last):
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 367, in synthesize_loop
    synthesize_inner_loop(fork, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 411, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 657, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 506, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 637, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    synthesize_loop_single_pr(toolbox, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 401, in synthesize_loop_single_pr
    synthesize_inner_loop(toolbox, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 411, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/92c16bc4-0b39-465b-bd2e-37e0a8400a9f/targets/github%2Fsynthtool;config=default/tests;query=nodejs-pubsub;failed=false).

Autosynth is still having trouble generating nodejs-pubsub. :sob:

Here's the output from running `synth.py`:

```
..

> @google-cloud/pubsub@2.3.0 compile /home/kbuilder/.cache/synthtool/nodejs-pubsub
> tsc -p . && cp -r protos build/

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m


Found 22 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/pubsub@2.3.0 compile: `tsc -p . && cp -r protos build/`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/pubsub@2.3.0 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-03T16_51_52_535Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/pubsub@2.3.0 prepare: `npm run compile-protos && npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/pubsub@2.3.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-03T16_51_52_561Z-debug.log
2020-08-03 09:51:52,577 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
2020-08-03 09:51:52,597 synthtool [DEBUG] > Wrote metadata to synth.metadata.
DEBUG:synthtool:Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-pubsub/synth.py", line 25, in <module>
    node.postprocess_gapic_library()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 193, in postprocess_gapic_library
    install(hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-08-03 09:51:52,647 autosynth [ERROR] > Synthesis failed
2020-08-03 09:51:52,647 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 901b355 chore(node): fix kokoro build path for cloud-rad (#1073)
2020-08-03 09:51:52,672 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-08-03 09:51:52,692 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing node_modules/
Traceback (most recent call last):
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 367, in synthesize_loop
    synthesize_inner_loop(fork, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 411, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 657, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 506, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 637, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    synthesize_loop_single_pr(toolbox, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 401, in synthesize_loop_single_pr
    synthesize_inner_loop(toolbox, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 411, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/eedccaac-978c-4f7b-b3e6-afa6069cbb20/targets/github%2Fsynthtool;config=default/tests;query=nodejs-pubsub;failed=false).

Autosynth is still having trouble generating nodejs-pubsub. :sob:

Here's the output from running `synth.py`:

```
..

> @google-cloud/pubsub@2.3.0 compile /home/kbuilder/.cache/synthtool/nodejs-pubsub
> tsc -p . && cp -r protos build/

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m


Found 22 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/pubsub@2.3.0 compile: `tsc -p . && cp -r protos build/`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/pubsub@2.3.0 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-04T16_50_30_302Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/pubsub@2.3.0 prepare: `npm run compile-protos && npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/pubsub@2.3.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-04T16_50_30_324Z-debug.log
2020-08-04 09:50:30,338 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
2020-08-04 09:50:30,357 synthtool [DEBUG] > Wrote metadata to synth.metadata.
DEBUG:synthtool:Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-pubsub/synth.py", line 25, in <module>
    node.postprocess_gapic_library()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 193, in postprocess_gapic_library
    install(hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-08-04 09:50:30,405 autosynth [ERROR] > Synthesis failed
2020-08-04 09:50:30,405 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 901b355 chore(node): fix kokoro build path for cloud-rad (#1073)
2020-08-04 09:50:30,430 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-08-04 09:50:30,450 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing node_modules/
Traceback (most recent call last):
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 367, in synthesize_loop
    synthesize_inner_loop(fork, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 411, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 657, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 506, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 637, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    synthesize_loop_single_pr(toolbox, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 401, in synthesize_loop_single_pr
    synthesize_inner_loop(toolbox, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 411, in synthesize_inner_loop
    synthesizer, len(toolbox.versions) - 1
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/ede1d3f6-026a-4ad6-ab3b-e21b79002e00/targets/github%2Fsynthtool;config=default/tests;query=nodejs-pubsub;failed=false).

Autosynth is still having trouble generating nodejs-pubsub. :sob:

Here's the output from running `synth.py`:

```
ookie=197337, name=package-lock.json.2116954433>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/protobufjs/cli/package-lock.json', wd=1354, mask=IN_MOVED_TO, cookie=197337, name=package-lock.json>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./protos/protos.json', wd=5, mask=IN_MODIFY, cookie=0, name=protos.json>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./protos/protos.js', wd=5, mask=IN_MODIFY, cookie=0, name=protos.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./protos/protos.d.ts', wd=5, mask=IN_MODIFY, cookie=0, name=protos.d.ts>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./protos/protos.d.ts', wd=5, mask=IN_MODIFY, cookie=0, name=protos.d.ts>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./protos/protos.d.ts', wd=5, mask=IN_MODIFY, cookie=0, name=protos.d.ts>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./protos/protos.d.ts', wd=5, mask=IN_MODIFY, cookie=0, name=protos.d.ts>

> @google-cloud/pubsub@2.3.0 precompile /home/kbuilder/.cache/synthtool/nodejs-pubsub
> gts clean

version: 12
Removing build ...

> @google-cloud/pubsub@2.3.0 compile /home/kbuilder/.cache/synthtool/nodejs-pubsub
> tsc -p . && cp -r protos build/

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m


Found 22 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/pubsub@2.3.0 compile: `tsc -p . && cp -r protos build/`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/pubsub@2.3.0 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-05T13_11_43_636Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/pubsub@2.3.0 prepare: `npm run compile-protos && npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/pubsub@2.3.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-05T13_11_43_699Z-debug.log
2020-08-05 06:11:43,733 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
2020-08-05 06:11:43,753 synthtool [DEBUG] > Wrote metadata to synth.metadata.
DEBUG:synthtool:Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-pubsub/synth.py", line 25, in <module>
    node.postprocess_gapic_library()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 193, in postprocess_gapic_library
    install(hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-08-05 06:11:43,811 autosynth [ERROR] > Synthesis failed
2020-08-05 06:11:43,811 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 901b355 chore(node): fix kokoro build path for cloud-rad (#1073)
2020-08-05 06:11:43,838 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-08-05 06:11:43,859 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/7b12b27f-67f6-4620-8ebe-674a90d07262/targets/github%2Fsynthtool;config=default/tests;query=nodejs-pubsub;failed=false).

Autosynth is still having trouble generating nodejs-pubsub. :sob:

Here's the output from running `synth.py`:

```
r/a732f932c2cbeb7e37e1543f189a2a73/external/gapic_generator_typescript/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
 (error An unexpected error occurred: "https://registry.yarnpkg.com/@protobufjs/inquire/-/inquire-1.1.0.tgz: Request failed \"503 Service Unavailable\"".
)
ERROR: /home/kbuilder/.cache/synthtool/googleapis/google/pubsub/v1/BUILD.bazel:232:1: //google/pubsub/v1:pubsub_nodejs_gapic depends on @gapic_generator_typescript//:protoc_plugin in repository @gapic_generator_typescript which failed to fetch. no such package '@npm//@bazel/typescript': yarn_install failed: yarn install v1.19.1
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
info If you think this is a bug, please open a bug report with the information provided in "/home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/gapic_generator_typescript/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
 (error An unexpected error occurred: "https://registry.yarnpkg.com/@protobufjs/inquire/-/inquire-1.1.0.tgz: Request failed \"503 Service Unavailable\"".
)
ERROR: Analysis of target '//google/pubsub/v1:pubsub-v1-nodejs' failed; build aborted: no such package '@npm//@bazel/typescript': yarn_install failed: yarn install v1.19.1
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
info If you think this is a bug, please open a bug report with the information provided in "/home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/gapic_generator_typescript/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
 (error An unexpected error occurred: "https://registry.yarnpkg.com/@protobufjs/inquire/-/inquire-1.1.0.tgz: Request failed \"503 Service Unavailable\"".
)
INFO: Elapsed time: 24.690s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (24 packages loaded, 600 targets configured)
FAILED: Build did NOT complete successfully (24 packages loaded, 600 targets configured)

ERROR:synthtool:Failed executing bazel --max_idle_secs=240 build //google/pubsub/v1:pubsub-v1-nodejs:

Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
Loading: 0 packages loaded
Loading: 0 packages loaded
Analyzing: target //google/pubsub/v1:pubsub-v1-nodejs (1 packages loaded, 0 targets configured)
Analyzing: target //google/pubsub/v1:pubsub-v1-nodejs (14 packages loaded, 25 targets configured)
Analyzing: target //google/pubsub/v1:pubsub-v1-nodejs (19 packages loaded, 132 targets configured)
Analyzing: target //google/pubsub/v1:pubsub-v1-nodejs (24 packages loaded, 600 targets configured)
INFO: Call stack for the definition of repository 'npm' which is a yarn_install (rule definition at /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/build_bazel_rules_nodejs/internal/npm_install/npm_install.bzl:411:16):
 - <builtin>
 - /home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/build_bazel_rules_nodejs/index.bzl:87:5
 - /home/kbuilder/.cache/synthtool/googleapis/WORKSPACE:293:1
Analyzing: target //google/pubsub/v1:pubsub-v1-nodejs (24 packages loaded, 600 targets configured)
ERROR: An error occurred during the fetch of repository 'npm':
   yarn_install failed: yarn install v1.19.1
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
info If you think this is a bug, please open a bug report with the information provided in "/home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/gapic_generator_typescript/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
 (error An unexpected error occurred: "https://registry.yarnpkg.com/@protobufjs/inquire/-/inquire-1.1.0.tgz: Request failed \"503 Service Unavailable\"".
)
ERROR: /home/kbuilder/.cache/synthtool/googleapis/google/pubsub/v1/BUILD.bazel:232:1: //google/pubsub/v1:pubsub_nodejs_gapic depends on @gapic_generator_typescript//:protoc_plugin in repository @gapic_generator_typescript which failed to fetch. no such package '@npm//@bazel/typescript': yarn_install failed: yarn install v1.19.1
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
info If you think this is a bug, please open a bug report with the information provided in "/home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/gapic_generator_typescript/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
 (error An unexpected error occurred: "https://registry.yarnpkg.com/@protobufjs/inquire/-/inquire-1.1.0.tgz: Request failed \"503 Service Unavailable\"".
)
ERROR: Analysis of target '//google/pubsub/v1:pubsub-v1-nodejs' failed; build aborted: no such package '@npm//@bazel/typescript': yarn_install failed: yarn install v1.19.1
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
info If you think this is a bug, please open a bug report with the information provided in "/home/kbuilder/.cache/bazel/_bazel_kbuilder/a732f932c2cbeb7e37e1543f189a2a73/external/gapic_generator_typescript/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
 (error An unexpected error occurred: "https://registry.yarnpkg.com/@protobufjs/inquire/-/inquire-1.1.0.tgz: Request failed \"503 Service Unavailable\"".
)
INFO: Elapsed time: 24.690s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (24 packages loaded, 600 targets configured)
FAILED: Build did NOT complete successfully (24 packages loaded, 600 targets configured)

2020-08-06 04:25:53,310 synthtool [DEBUG] > Wrote metadata to synth.metadata.
DEBUG:synthtool:Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-pubsub/synth.py", line 15, in <module>
    library = gapic.node_library('pubsub', version, proto_path=f'google/pubsub/{version}')
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 52, in node_library
    return self._generate_code(service, version, "nodejs", **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_bazel.py", line 183, in _generate_code
    shell.run(bazel_run_args)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['bazel', '--max_idle_secs=240', 'build', '//google/pubsub/v1:pubsub-v1-nodejs']' returned non-zero exit status 1.
2020-08-06 04:25:53,359 autosynth [ERROR] > Synthesis failed
2020-08-06 04:25:53,359 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 901b355 chore(node): fix kokoro build path for cloud-rad (#1073)
2020-08-06 04:25:53,365 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-08-06 04:25:53,371 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/76bb7f6f-4d47-4888-b0d2-a8761a276cc8/targets/github%2Fsynthtool;config=default/tests;query=nodejs-pubsub;failed=false).

Autosynth is still having trouble generating nodejs-pubsub. :sob:

Here's the output from running `synth.py`:

```
ookie=196527, name=package-lock.json.3134687719>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/protobufjs/cli/package-lock.json', wd=1380, mask=IN_MOVED_TO, cookie=196527, name=package-lock.json>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./protos/protos.json', wd=5, mask=IN_MODIFY, cookie=0, name=protos.json>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./protos/protos.js', wd=5, mask=IN_MODIFY, cookie=0, name=protos.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./protos/protos.d.ts', wd=5, mask=IN_MODIFY, cookie=0, name=protos.d.ts>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./protos/protos.d.ts', wd=5, mask=IN_MODIFY, cookie=0, name=protos.d.ts>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./protos/protos.d.ts', wd=5, mask=IN_MODIFY, cookie=0, name=protos.d.ts>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./protos/protos.d.ts', wd=5, mask=IN_MODIFY, cookie=0, name=protos.d.ts>

> @google-cloud/pubsub@2.3.0 precompile /home/kbuilder/.cache/synthtool/nodejs-pubsub
> gts clean

version: 12
Removing build ...

> @google-cloud/pubsub@2.3.0 compile /home/kbuilder/.cache/synthtool/nodejs-pubsub
> tsc -p . && cp -r protos build/

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m


Found 22 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/pubsub@2.3.0 compile: `tsc -p . && cp -r protos build/`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/pubsub@2.3.0 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-06T19_52_21_129Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/pubsub@2.3.0 prepare: `npm run compile-protos && npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/pubsub@2.3.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-06T19_52_21_198Z-debug.log
2020-08-06 12:52:21,230 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
2020-08-06 12:52:21,249 synthtool [DEBUG] > Wrote metadata to synth.metadata.
DEBUG:synthtool:Wrote metadata to synth.metadata.
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-pubsub/synth.py", line 25, in <module>
    node.postprocess_gapic_library()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 193, in postprocess_gapic_library
    install(hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-08-06 12:52:21,305 autosynth [ERROR] > Synthesis failed
2020-08-06 12:52:21,306 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 901b355 chore(node): fix kokoro build path for cloud-rad (#1073)
2020-08-06 12:52:21,330 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-08-06 12:52:21,350 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/374d348d-8a6c-4e4c-b1a2-7114e0aa1049/targets/github%2Fsynthtool;config=default/tests;query=nodejs-pubsub;failed=false).

Autosynth is still having trouble generating nodejs-pubsub. :sob:

Here's the output from running `synth.py`:

```
r:in-event <InotifyEvent: src_path=b'./node_modules/protobufjs/cli/package-lock.json.3080646590', wd=1375, mask=IN_MOVED_FROM, cookie=196422, name=package-lock.json.3080646590>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./node_modules/protobufjs/cli/package-lock.json', wd=1375, mask=IN_MOVED_TO, cookie=196422, name=package-lock.json>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./protos/protos.json', wd=5, mask=IN_MODIFY, cookie=0, name=protos.json>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./protos/protos.js', wd=5, mask=IN_MODIFY, cookie=0, name=protos.js>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./protos/protos.d.ts', wd=5, mask=IN_MODIFY, cookie=0, name=protos.d.ts>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./protos/protos.d.ts', wd=5, mask=IN_MODIFY, cookie=0, name=protos.d.ts>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./protos/protos.d.ts', wd=5, mask=IN_MODIFY, cookie=0, name=protos.d.ts>
DEBUG:watchdog.observers.inotify_buffer:in-event <InotifyEvent: src_path=b'./protos/protos.d.ts', wd=5, mask=IN_MODIFY, cookie=0, name=protos.d.ts>

> @google-cloud/pubsub@2.3.0 precompile /home/kbuilder/.cache/synthtool/nodejs-pubsub
> gts clean

version: 12
Removing build ...

> @google-cloud/pubsub@2.3.0 compile /home/kbuilder/.cache/synthtool/nodejs-pubsub
> tsc -p . && cp -r protos build/

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/fallback.d.ts[0m:[93m119[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m119[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m   [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1005: [0m'from' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m10[0m - [91merror[0m[90m TS1141: [0mString literal expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m         ~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m13[0m - [91merror[0m[90m TS2304: [0mCannot find name 'protobufMinimal'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m            ~~~~~~~~~~~~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m29[0m - [91merror[0m[90m TS2304: [0mCannot find name 'from'.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                            ~~~~[0m

[96mnode_modules/google-gax/build/src/index.d.ts[0m:[93m47[0m:[93m34[0m - [91merror[0m[90m TS1005: [0m';' expected.

[7m47[0m export * as protobufMinimal from 'protobufjs/minimal';
[7m  [0m [91m                                 ~~~~~~~~~~~~~~~~~~~~[0m


Found 22 errors.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/pubsub@2.3.0 compile: `tsc -p . && cp -r protos build/`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/pubsub@2.3.0 compile script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-07T12_49_51_467Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @google-cloud/pubsub@2.3.0 prepare: `npm run compile-protos && npm run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @google-cloud/pubsub@2.3.0 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kbuilder/.npm/_logs/2020-08-07T12_49_51_536Z-debug.log
2020-08-07 05:49:51,565 synthtool [ERROR] > Failed executing npm install:

None
ERROR:synthtool:Failed executing npm install:

None
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/nodejs-pubsub/synth.py", line 25, in <module>
    node.postprocess_gapic_library()
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 193, in postprocess_gapic_library
    install(hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/languages/node.py", line 167, in install
    shell.run(["npm", "install"], hide_output=hide_output)
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 39, in run
    raise exc
  File "/tmpfs/src/github/synthtool/synthtool/shell.py", line 33, in run
    encoding="utf-8",
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 438, in run
    output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['npm', 'install']' returned non-zero exit status 1.
2020-08-07 05:49:51,638 autosynth [ERROR] > Synthesis failed
2020-08-07 05:49:51,639 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 901b355 chore(node): fix kokoro build path for cloud-rad (#1073)
2020-08-07 05:49:51,663 autosynth [DEBUG] > Running: git checkout autosynth
Switched to branch 'autosynth'
2020-08-07 05:49:51,683 autosynth [DEBUG] > Running: git clean -fdx
Removing __pycache__/
Removing node_modules/
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 690, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 539, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 670, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 375, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 273, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/46f39e56-2d48-4dfc-8fcc-e5cf5f484e53/targets/github%2Fsynthtool;config=default/tests;query=nodejs-pubsub;failed=false).

--- a/package.json
+++ b/package.json
@@ -93,7 +93,7 @@
     "sinon": "^9.0.0",
     "tmp": "^0.2.0",
     "ts-loader": "^8.0.0",
-    "typescript": "3.6.4",
+    "typescript": "^3.8.3",
     "uuid": "^8.0.0",
     "webpack": "^4.42.0",
     "webpack-cli": "^3.3.11",

--- a/test/message-stream.ts
+++ b/test/message-stream.ts
@@ -69,7 +69,7 @@ class FakePassThrough extends PassThrough {
     this.options = options;
   }
   destroy(err?: Error): void {
-    if (super.destroy) {
+    if (typeof super.destroy === 'function') {
       return super.destroy(err);
     }
     destroy(this, err);
@@ -96,7 +96,7 @@ class FakeGrpcStream extends Duplex {
     });
   }
   destroy(err?: Error): void {
-    if (super.destroy) {
+    if (typeof super.destroy === 'function') {
       return super.destroy(err);
     }
     destroy(this, err);

--- a/test/pubsub.ts
+++ b/test/pubsub.ts
@@ -307,11 +307,12 @@ describe('PubSub', () => {
       name: 'subscription-name',
     };
 
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const subClass = Subscription as any;
+
     beforeEach(() => {
-      ((Subscription as {}) as typeof subby.Subscription).formatMetadata_ = (
-        metadata: subby.SubscriptionMetadata
-      ) => {
-        return Object.assign({}, metadata) as subby.SubscriptionMetadata;
+      subClass.formatMetadata_ = (metadata: {}) => {
+        return Object.assign({}, metadata);
       };
     });
 
@@ -469,11 +470,9 @@ describe('PubSub', () => {
         a: 'a',
       };
 
-      ((Subscription as {}) as typeof subby.Subscription).formatMetadata_ = (
-        metadata: subby.SubscriptionMetadata
-      ) => {
+      subClass.formatMetadata_ = (metadata: {}) => {
         assert.deepStrictEqual(metadata, fakeMetadata);
-        return (formatted as {}) as subby.SubscriptionMetadata;
+        return formatted;
       };
 
       pubsub.request = (config: pubsubTypes.RequestConfig) => {

--- a/test/subscription.ts
+++ b/test/subscription.ts
@@ -919,8 +919,11 @@ describe('Subscription', () => {
       pushEndpoint: 'http://noop.com/push',
     };
 
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const subClass = subby.Subscription as any;
+
     beforeEach(() => {
-      Subscription.formatMetadata_ = (metadata: subby.SubscriptionMetadata) => {
+      subClass.formatMetadata_ = (metadata: {}) => {
         return Object.assign({}, metadata);
       };
     });

